extensions/vscode/src/utils/config.ts

```diff
@@ -0,0 +1,47 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+function getConfigFile(configFilePath: string): string | null {
+  const configDirs = process.env.XDG_CONFIG_DIRS ? 
+         process.env.XDG_CONFIG_DIRS.split(':') : 
+         ['/etc'];
```

Comment:
Will this work on Windows machines or cause a file failure? I'm not sure if we need to use `path.sep` from the `path` module or something else.

Reply:
I don't think we use this file on windows or mac so I made it linux only with an explicit platform check. Thanks also for catching that!